test/Fcs_load_and_transform.jl

```diff
@@ -8,11 +8,17 @@ FCS, we need to see what functionality is missing and
 extend this in the original package
 =#
 
+using Distributed
+
+# p = addprocs(2)
+#
+# @everywhere using GigaSOM
+
 using GigaSOM
 using CSV
 using DataFrames
 
-dataPath = "data/"
+dataPath = "/home/ohunewald/work/GigaSOM_data/PBMC8_fcs_files"
```

Comment:
May you set here the data path relative? i.e `test/data`?

Reply:
done